hera_cal/abscal.py

```diff
@@ -881,13 +883,17 @@ def abscal_run(data_files, model_files, verbose=True, overwrite=False, write_cal
                 a list of paths to miriad file(s) containing complex
                 visibility data, or a path itself
 
+    calfits_files : type=list or string, path(s) to calfits files(s)
```

Comment:
When I first read this, I thought you were supporting multiple calfits files that get multiplied together to make a single calfits file, but actually you're only supporting a single calfits file to multiply in _per output file_. That's totally fine, but it could use clarification in the documentation.

Relatedly, you probably want to say here that if a calfits file is provided, quals, total quals, and history get passed through.